tasks/unit_tests/components_tests.py

```diff
@@ -11,12 +11,21 @@ class TestComponents(unittest.TestCase):
     def setUp(self):
         # Create a temporary directory of the source directory to allow
         # tests to modify the source files
-        # TODO: Add tests that modify source files to ensure that errors work
         self.tmpdir = tempfile.mkdtemp()
-        shutil.copytree('tasks/unit_tests/testdata/components_src/comp', os.path.join(self.tmpdir, 'comp'))
-        # Preserve currenty directory, change to temp directory
         self.origDir = os.getcwd()
+        self.copy_component_src_to_tmpdir()
+        # Preserve currenty directory, change to temp directory
         os.chdir(self.tmpdir)
+        classicComp = 'comp/classic/classicimpl'
+        if classicComp not in components.components_classic_style:
+            components.components_classic_style.append(classicComp)
```

Comment:
Should we add a tear down definition that would remove the `'comp/classic/classicimpl'` at the end of the tests?

Reply:
Sure, done.